### PR TITLE
Restructure package

### PR DESCRIPTION
**Pull request by DavidGuamanDavila**
# Problem
- The original file structure was inconvenient to navigate and launch
# Solution
- Restructure package to traditional package format of Iceberg ASV
# Testing
image
The launch file was tested and everything worked as expected

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.0.2)
-project(yolov5_ros)
+project(objectdetect_yolov5ros_pkg)
```
Comment (gpearcey) [inline CMakeLists.txt]: Could we rename this package to something shorter? Maybe "ros_obj_detection"? or Just leave it as "yolov5_ros"?

File: launch/sim_yolo_v5.launch
Comment (gpearcey) [inline launch/sim_yolo_v5.launch:1]: Rename file to `sim_<package_name>.launch`

File: launch/yolo_v5.launch
Comment (gpearcey) [inline launch/yolo_v5.launch:1]: Rename file to `<package_name>.launch`

**Review by gpearcey — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (DavidGuamanDavila) [reply CMakeLists.txt]: Sounds good! I'll do that!

**Review by gpearcey — APPROVED** [review]

**Pull request merged**